tests/integration/tests/test_version_upgrades.py

```diff
@@ -12,40 +12,45 @@
 
 @pytest.mark.node_count(1)
 @pytest.mark.no_setup()
-@pytest.mark.xfail(reason="cilium failures are blocking this from working")
```

Comment:
@bschimke95 I did not see this to be the case any longer.  I've fixed this test again and it seems to pass??

Reply:
yes, I think Berkay implemented a fix.